MOBILE_CLIENT/IOS/_HIGH/UNSAFE_KEYCHAIN_ACCESSIBILITY/meta.json

```
@@ -0,0 +1,27 @@
+{
+  "risk_rating": "high",
+  "short_description": "Insecure Storage: Unsafe Keychain Accessibility vulnerability allows sensitive data stored on the device to be easily accessed by malicious actors.",
+  "references": {
+    "Apple documentation: Restricting keychain item accessibility": "https://developer.apple.com/documentation/security/keychain_services/keychain_items/restricting_keychain_item_accessibility",
+    "Secure Mobile Biometric Authentication": "https://blog.ostorlab.co/secure-mobile-biometric-authentication.html"
+  },
+  "title": "Insecure Storage: Unsafe Keychain Accessibility",
```

```suggestion
  "title": "Insecure Keychain Storage",
```
Simple is better than complicated.
Complicated is better than complex. ;)